F/FlexLexer/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+using BinaryBuilder
+
+# Collection of sources required to build Gettext
+name = "FlexLexer"
+version = v"2.6.4"
+
+sources = [
+            FileSource("https://raw.githubusercontent.com/westes/flex/master/src/FlexLexer.h",
+                       "ee9859d6b3027ed565f98f42744e438ab31b2cd2e9f797ddf870029ca2021686"),
+            FileSource("https://raw.githubusercontent.com/westes/flex/master/COPYING",
```

Comment:
Even if we were going with this PR (and I'm very sceptical about it), these URLs are not acceptable: you can't reference `master`, it's a moving target, you need a permanent link pointing to a specific commit